system/ui/spinner.py

```diff
@@ -41,8 +41,10 @@ def main():
   comma_position = rl.Vector2(center.x - TEXTURE_SIZE / 2.0, center.y - TEXTURE_SIZE / 2.0)
 
   for _ in gui_app.render():
```

Comment:
is it worth putting an FPS rate keeper in gui_app to ensure consistent animations across all raylib apps we create?

Reply:
Raylib keeps a consistent FPS for us. I use frequency to adjust the spinner's speed so it works correctly even if the FPS changes from the default 60.